lib/crowbar/client/command/upgrade/prechecks.rb

```diff
@@ -66,26 +66,32 @@ def content_from(request)
               request.parsed_response["checks"].each do |check_id, values|
                 # make the check_id server agnostic
                 # the check_id could be named differently in the server response
-                check_id = values["errors"].keys.first if values["errors"].any?
+                check_ids = values["errors"].keys if values["errors"].any?
```

Comment:
If key in errors for given precheck is the same as the key under this precheck's `errors` hash then why do we have this PR? Ha! Gotcha! :o)